src/components/Header/header.module.css

```diff
@@ -0,0 +1,51 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+code {
+    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+        monospace;
+}
+header {
```

Comment:
The header should be sticky, I don't see it in CSS, if it's not implemented please update it